ts/src/index.ts

```diff
@@ -122,21 +163,10 @@ let profiler: Profiler|undefined = undefined;
  *
  */
 export async function start(config: Config = {}): Promise<void> {
-  if (!semver.satisfies(process.version, pjson.engines.node)) {
-    logError(
-        `Could not start profiler: node version ${process.version}` +
-            ` does not satisfies "${pjson.engines.node}"`,
-        config);
-    return;
-  }
-  let normalizedConfig: ProfilerConfig;
-  try {
-    normalizedConfig = await initConfig(config);
-  } catch (e) {
-    logError(`Could not start profiler: ${e}`, config);
+  const profiler: Profiler|undefined = await createProfiler(config);
+  if (profiler === undefined) {
```

Comment:
this test can be removed if you throw above. Omit the type for the local variable `profiler`.

Reply:
done